webpack.config.js

```diff
@@ -29,10 +29,17 @@ babelConf.babelrc = false; // disabel babelrc reading, as we've just done it
 const replacementPluginConf = babelConf.plugins.find(plugin => Array.isArray(plugin) && plugin[0] === 'minify-replace');
 replacementPluginConf[1].replacements.find(decl => decl.identifierName === '__DEBUG__').replacement.value = debugBuild;
 
+const include = [
+    path.resolve(__dirname, 'src'),
+    path.resolve(__dirname, 'test'),
+    path.resolve(__dirname, 'utils'),
+];
+
 module.exports = {
+    context: path.resolve(__dirname),
```

Comment:
Weird from webpack that we can't use this context everywhere :thinking:

Reply:
some options need absolute path...